prac_07/project.py

```diff
@@ -15,4 +15,8 @@ def __init__(self, name, start_date, priority, cost_estimate, completion_percent
         self.completion_percentage = completion_percentage
 
     def __str__(self):
-        return f"{self.name}--{self.start_date}--{self.priority}--{self.cost_estimate}--{self.completion_percentage}"
+        return f"{self.name}, start: {self.start_date}, priority {self.priority}, estimate: ${self.cost_estimate}, " \
+               f"completion: {self.completion_percentage}%"
+
+    def __lt__(self, other):
+        return self.priority > other
```

Comment:
Should this be < instead of > as the smaller priority number means that it is of higher importance (e.g. '1' is top priority)?